packages/mobile/src/screens/sign-on-screen/screens/CreatePasswordScreen.tsx

```diff
@@ -58,34 +61,48 @@ export const CreatePasswordScreen = () => {
     >
       {({ handleSubmit: triggerSubmit, dirty, isValid }) => (
         <Page>
-          <Heading
-            heading={messages.createYourPassword}
-            description={messages.description}
-          />
-          <Flex direction='column' h='100%' gap='l'>
-            <ReadOnlyField label={messages.yourEmail} value={email} />
-            <TextField
-              name='password'
-              label={messages.passwordLabel}
-              textContentType='password'
-              secureTextEntry
-              noGutter
+          <KeyboardAvoidingView
+            keyboardShowingOffset={keyboardOffset - 16}
+            style={{ overflow: 'hidden' }}
+          >
+            <Heading
+              heading={messages.createYourPassword}
+              description={messages.description}
             />
-            <TextField
-              name='confirmPassword'
-              label={messages.confirmPasswordLabel}
-              textContentType='password'
-              secureTextEntry
-              noGutter
+            <Flex
+              direction='column'
+              h='100%'
+              gap='l'
+              onLayout={(e) => {
```

Comment:
nifty! so this is to get the keyboard to go specifically to this view rather than just go up some arbitrary amount?

Reply:
yea exactly